builder.go

```diff
@@ -116,6 +116,11 @@ func (b Builder) Build(ctx context.Context, outputFile string) error {
 			"-trimpath",
 		)
 	}
+	if b.Debug {
```

Comment:
```suggestion
	} else {
```

And actually, maybe we want to invert the `if` statement to be positive: `if b.Debug { ... } else { ... }` just so it reads easier.

Reply:
I'm confused @mholt, it's already "positive" :thinking:

Reply:
I mean, instead of

```go
 	if !b.Debug {
		cmd.Args = append(cmd.Args,
			"-ldflags", "-w -s", // trim debug symbols
			"-trimpath",
		)
	} else {
		cmd.Args = append(cmd.Args,
			 "-gcflags", "all=-N -l", // support dlv
			)
	}
```

do this:


```go
 	if b.Debug {
		cmd.Args = append(cmd.Args,
			"-gcflags", "all=-N -l", // support dlv
		)
	} else {
		cmd.Args = append(cmd.Args,
			"-ldflags", "-w -s", // trim debug symbols
			"-trimpath",
		)
	}
```

Reply:
(I deleted my comment, sorry I misread the diff/suggested change cause you were commenting on an unchanged line)

Reply:
Good, so what should I do? Make new changes then push again?
Cause I'm new to PR conversation, I have never use this github feature before.

Reply:
That's correct. You could even make the changes on github itself by editing the file here.

Reply:
Ideally, change it on your computer and test the change, then push to your branch again. This will automatically update. :+1:

Reply:
ok I push the latest changes